apps/meteor/tests/e2e/file-upload.spec.ts

```diff
@@ -54,4 +63,33 @@ test.describe.serial('file-upload', () => {
 		await expect(poHomeChannel.content.getFileDescription).toHaveText('lst_description');
 		await expect(poHomeChannel.content.lastMessageFileName).toContainText('lst-test.lst');
 	});
+
+	test('expect not to send drawio file when the "Reject unknown media types" setting is enabled', async ({ api, page }) => {
```

Comment:
It would be better to use the setting name here since we have nothing mentioning this "Reject...", it could cause confusion